CoPla/.htaccess

```diff
@@ -0,0 +1,48 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .jsonld
+# Rewrite engine setup
+RewriteEngine On
+#Change the path to the folder here
+RewriteBase https://github.com/kai-vu/companion-planting-decision-support/tree/main/owl/Companion%20Planting%20Ontology
```

Comment:
I'm pretty sure this feature is for local path rewrites and doesn't work with arbitrary URLs.  Did you test these rules?